sql-bricks.js

```diff
@@ -53,11 +53,10 @@
 
     // shift the placeholder indexes if there are already values
     if (opts.value_ix > 1) {
-      var orig_val_ix = opts.value_ix;
       if (opts.placeholder == '$%d')
-        str = str.replace(/\$(\d+)/g, function(match, capture) { opts.value_ix++; return '$' + (parseInt(capture, 10) + orig_val_ix - 1); });
+        str = str.replace(/\$(\d+)/g, function(match, capture) { return '$' + (parseInt(capture, 10) + opts.value_ix - 1); });
       else if (opts.placeholder == '?%d')
-        str = str.replace(/\?(\d+)/g, function(match, capture) { opts.value_ix++; return '?' + parseInt(capture, 10) + orig_val_ix - 1; });
+        str = str.replace(/\?(\d+)/g, function(match, capture) { return '?' + parseInt(capture, 10) + opts.value_ix - 1; });

```

Comment:
I think it's definitely better because it's at least correct for sql('a = $1 or b = $1 or c = $2', 123, 234) - in this case there are 2 values, but 3 replacements